src/test/java/org/jenkinsci/plugins/logparser/LogParserWorkflowTest.java

```diff
@@ -0,0 +1,54 @@
+package org.jenkinsci.plugins.logparser;
+
+import hudson.FilePath;
+import hudson.plugins.logparser.LogParserAction;
+import hudson.plugins.logparser.LogParserPublisher;
+import hudson.tasks.Maven;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * In this test suite we initialize the Job workspaces with a resource (maven-project1.zip) that contains a Maven
+ * project.
+ */
+public class LogParserWorkflowTest {
+
+    @ClassRule
+    public static JenkinsRule jenkinsRule = new JenkinsRule();
```

Comment:
No purpose in using `@ClassRule` (rather than the usual `@Rule`) when you have only one test case!